ramls/instance-bulk.raml

```diff
@@ -0,0 +1,50 @@
+#%RAML 1.0
+title: Inventory Storage Instance Bulk API
+version: v0.1
+protocols: [ HTTP, HTTPS ]
+baseUri: http://localhost
+
+documentation:
+  - title: Inventory Storage Instance Bulk Download API
+    content: <b>API for downloading a bulk set of Instance IDs</b>
+
+types:
+  id: !include id.json
```

Comment:
I don't see where the type is used nor the schema itself.

Reply:
It was used up until here: https://github.com/folio-org/mod-inventory-storage/pull/405/commits/9e8151e67207a15c5c2ef7cfa8cd2f998704e3a5

I seem to have missed removing this one.